.dockerignore

```diff
@@ -0,0 +1,3 @@
+.env
+.git
+.vscode
```

Comment:
Not super important but you might also want to consider ignoring `**/__pycache__`
- note the `**` is useful as unlike `.gitignore`, `filename` alone only ignores that file in the root, not in all sub-directories. So e.g. if someone placed the `.env` file in `todo_app` folder here, that file could still be copied in